src/main/scala/mesosphere/marathon/api/v2/Validation.scala

```diff
@@ -91,6 +93,25 @@ object Validation {
     }
   }
 
+  def fetchUriHasSupportedProtocol: Validator[FetchUri] = {
+    lazy val supportedProtocols = Set("http", "https", "ftp", "ftps", "hdfs")
+
+    new Validator[FetchUri] {
+      def apply(uri: FetchUri) = {
+        Try {
+          val value = new URL(uri.uri)
```

Comment:
Current implementation allows simple  files, ftp, s3 etc.
There is also a plugin mechanism to extend that functionality.
So: I don't think we can validate here.